src/actors/FilterBarActor.js

```diff
@@ -0,0 +1,88 @@
+import * as SharedUtils from '../utils/SharedUtils';
+
+export class FilterBarActor {
+  constructor(filterBarStore, tableStore) {
+    this.filterBarStore = filterBarStore;
+    this.tableStore = tableStore;
+  }
+
+  getSavedSearches() {
```

Comment:
Actors shouldn't query

Component -> Actor -> Store -> Component

Just query the store directly